staging/helmfile.yaml

```diff
@@ -1,5 +1,7 @@
 ---
 repositories:
+  - name: descheduler
```

Comment:
Do we need this on fakeprod too? (Dev I assume is a different animal)

Reply:
umm yeah fake can get it

Reply:
added fake to PR